kv_v1.go

```diff
@@ -94,3 +101,16 @@ func (k *KVv1) Delete(key string) error {
 
 	return nil
 }
+
+func (k *KVv1) mapError(err error) error {
```

Comment:
On second thought (I just noticed this affects the KV Engine), why would we need to map a StatusNotFound error to anything else? Besides breaking the existing API, mapping to a completely unrelated error doesn't seem useful. I think this should be kept as is, any consumer can check the StatusCode already?